machinist/mserver/command.go

```diff
@@ -61,5 +62,6 @@ func NewCommand(bf *client.BaseFlags) *cobra.Command {
 	c.PersistentFlags().IntVar(&cpf.DnsPort, "dns-port", 5353, "the udp port that the dns will be served on, also note it will also allocate the tcp socket on it as well")
 	c.PersistentFlags().StringSliceVar(&cpf.Domains, "domains", []string{}, "domains that the master ControlPlane will be serving")
 	c.PersistentFlags().StringVar(&cpf.BindNet, "bind-net", "127.0.0.1", "the address to bind the grpc listener to")
+	c.PersistentFlags().StringVar(&cpf.StateFile, "state", "./state.json", "file to write and load state to")
```

Comment:
Genuinely not sure if I should use os.Getcwd here

Reply:
I think this is fine. A system daemon will likely need to store the state file in `/var/lib/$nameofdaemon`, and if run in a docker container, be persisted through a volume.

The only thing I would do is use lib/config/ and the store interface here, but it can come in a separate PR.
(so, good as is, no action from me)